modules/quickstart/examples/preview-hello.mo

```diff
@@ -0,0 +1,17 @@
+import Nat "mo:base/Nat";
+
+var count: Nat = 0;
+var output : Text = "Welcome to the simplified developer preview, ";
+var name : Text = " ";
```

Comment:
It looks like this var `name` is never used (there is a separate name parameter to `whoami`)
```suggestion
```

Reply:
Oh yeah, I was thinking of trying to do something like this:
if myargname = whoami name, increment the count, else reset the count to 1, but I couldn't get the syntax right and abandoned ship